src/main/java/florizz/core/Parser.java

```diff
@@ -277,55 +277,61 @@ private static InfoCommand handleInfoCommand(String flowerName) {
      * @return The parsed occasion.
      * @throws FlorizzException If the input does not match the required format.
      */
-    public static String parseOccasion(String argument) throws FlorizzException{
+    public static boolean parseOccasion(String argument) {
         if (argument == null) {
-            throw new FlorizzException("No argument detected! " +
+            System.out.println("No argument detected! " +
                     "Please input an occasion");
+            return false;
         }
 
         if (!argument.matches(PARSE_OCCASION_REGEX)) {
-            throw new FlorizzException("Incorrect format detected! " +
+            System.out.println("Incorrect format detected! " +
                     "Please input a single occasion");
+            return false;
```

Comment:
It might be good if all `system.out` should be done by `Ui` only to follow single responsibility principle. I understand why you remove the florizz exception since it kinds of stop users from inputting more command.